app_dart/bin/generate_jspb.dart

```diff
@@ -52,20 +53,18 @@ String getLocalConfigContent(String path) {
 }
```

Comment:
getLocalFileContent should be deleted now that it's no longer used

Reply:
umm I added that part back in to pass an integration test that checks for local yaml validation. I was thinking of replacing the old integration test with a new test that validates against tip of tree, but then realized we would have to keep the jspb expectation file updated all the time, since tip of tree yaml file could change over time. Shall we remove the entire integration test that validates local yaml as well?